recipes-trik/trik-runtime/files/log_manager.sh

```diff
@@ -91,6 +91,8 @@ prepare_tmp_dir() {
 	tmp_dir_path="${archive_path}/${tmp_dir_name}"
 
 	mkdir -p "$tmp_dir_path"
+	
+	echo "${tmp_dir_path}"
```

Comment:
Please, remain here only echo and move special_echo after "compress" execution below.